Examples are broken

```
root@ubuntu:~# go get -u -x -v github.com/yobert/alsa/cmd/beep
cd /root/go/src/github.com/yobert/alsa
git config remote.origin.url
github.com/yobert/alsa (download)
cd /root/go/src/github.com/yobert/alsa
git pull --ff-only
cd /root/go/src/github.com/yobert/alsa
git submodule update --init --recursive
cd /root/go/src/github.com/yobert/alsa
git show-ref
cd /root/go/src/github.com/yobert/alsa
git submodule update --init --recursive
cd /root/go/src/github.com/yobert/alsa
git config remote.origin.url
cd /root/go/src/github.com/yobert/alsa
git config remote.origin.url
cd /root/go/src/github.com/yobert/alsa
git config remote.origin.url
cd /root/go/src/github.com/yobert/alsa
git config remote.origin.url
cd /root/go/src/github.com/yobert/alsa
git config remote.origin.url
WORK=/tmp/go-build325338770
github.com/yobert/alsa/alsatype
mkdir -p $WORK/b036/
cat >$WORK/b036/importcfg << 'EOF' # internal
# import config
packagefile fmt=/usr/lib/go-1.14/pkg/linux_arm64/fmt.a
packagefile reflect=/usr/lib/go-1.14/pkg/linux_arm64/reflect.a
EOF
cd /root/go/src/github.com/yobert/alsa/alsatype
/usr/lib/go-1.14/pkg/tool/linux_arm64/compile -o $WORK/b036/_pkg_.a -trimpath "$WORK/b036=>" -p github.com/yobert/alsa/alsatype -complete -buildid QkdTK6SZxXLGeNlwwVY9/QkdTK6SZxXLGeNlwwVY9 -goversion go1.14.7 -D "" -importcfg $WORK/b036/importcfg -pack -c=4 ./PVersion.go ./SwParams.go ./uint.go
# github.com/yobert/alsa/alsatype
go/src/github.com/yobert/alsa/alsatype/PVersion.go:5:9: undefined: PVersion
go/src/github.com/yobert/alsa/alsatype/PVersion.go:8:9: undefined: PVersion
go/src/github.com/yobert/alsa/alsatype/PVersion.go:11:9: undefined: PVersion
go/src/github.com/yobert/alsa/alsatype/PVersion.go:14:9: undefined: PVersion
go/src/github.com/yobert/alsa/alsatype/SwParams.go:8:10: undefined: SwParams
go/src/github.com/yobert/alsa/alsatype/SwParams.go:9:17: undefined: SwParams
go/src/github.com/yobert/alsa/alsatype/SwParams.go:12:10: undefined: SwParams
```

--- a/alsatype/types_arm64.go
+++ b/alsatype/types_arm64.go
@@ -1,0 +1,30 @@
+package alsatype
+
+type (
+	Uframes uint64 // snd_pcm_uframes_t
+	Sframes int64  // snd_pcm_sframes_t
+)
+
+type Timespec struct {
+	Sec  int
+	Nsec int
+}
+
+type SwParams struct {
+	TstampMode int32
+	PeriodStep uint32
+	SleepMin   uint32
+
+	AvailMin         Uframes
+	XferAlign        Uframes
+	StartThreshold   Uframes
+	StopThreshold    Uframes
+	SilenceThreshold Uframes
+	SilenceSize      Uframes
+	Boundary         Uframes
+
+	Proto      PVersion
+	TstampType uint32
+	Reserved   [56]byte
+}
+type PVersion uint32

